actionpack/lib/action_dispatch/routing/inspector.rb

```diff
@@ -59,13 +59,11 @@ def initialize(routes)
         @routes = routes
       end
 
-      def format(formatter, filter = nil)
-        routes_to_display = filter_routes(filter)
-
+      def format(formatter, **options)
+        routes_to_display = filter_routes(options)
         routes = collect_routes(routes_to_display)
-
         if routes.none?
-          formatter.no_routes(collect_routes(@routes), filter)
+          formatter.no_routes(collect_routes(@routes), options[:pattern])
```

Comment:
We still need to pass it for this- https://github.com/vipulnsward/rails/blob/212bf9233c8ec0dddfd3b97da8fc53cce83a6cd2/actionpack/lib/action_dispatch/routing/inspector.rb#L165-L167
and determining if controller exists or not.

To think about it, we can just always say - "No routes were found for specified controller"